basxconnect/mailer_integration/abstract/abstract_datasource.py

```diff
@@ -1,7 +1,18 @@
 import abc
 from typing import List, NamedTuple
 
-from basxconnect.mailer_integration.abstract.abstract_mailer_person import MailerPerson
+
+class MailerPerson(NamedTuple):
+    first_name: str
+    last_name: str
+    display_name: str
+    email: str
+    interests_ids: List[str]
+    status: str
+    country: str
+    postcode: str
+    address: str
+    city: str
```

Comment:
This could be described as the "minimal interface between basxconnect and any external contact-management system", right? I think that would be good, it is easy to understand what will get synchronized.
If we have other external systems in the future (mailer or otherwise) it will be valuable to reuse this.
Maybe add a comment what format "country" should be.

Reply:
For the future I think I see it as the "maximal" interface of which fields basxconnect can make use of. Not every mailer will probably have postal addresses...